resources/benchmark-runner.mjs

```diff
@@ -163,6 +163,166 @@ const WarmupSuite = {
     ],
 };
 
+class TimerCallbacks {
+    constructor(firstCallback, secondCallback) {
+        this._firstCallback = firstCallback;
+        this._secondCallback = secondCallback;
+        this._setTimeoutCallback = this._setTimeout.bind(this);
```

Comment:
Nit: I'd find it easier to follow these classes if property name was always `_setTimeoutCallback` both in the class declaration and in this statement reassigning to the bound function (`this._setTimeoutCallback = this._setTimeoutCallback.bind(this);`)